README_SKINNY.rst

```diff
@@ -13,4 +13,5 @@ Additional dependencies can be installed to leverage the full feature set of MLf
 
 * To use the `mlflow.sklearn` component of MLflow Models, install `scikit-learn`, `numpy`, and `pandas`.
 * To use SQL-based metadata storage, install `sqlalchemy`, `alembic`, and `sqlparse`.
+* To use serving-based features, install `flask`.
```

Comment:
Does `mlflow models serve` require pandas to be installed?

Reply:
? The flavor doesn't even import pandas.  Let me fix that real quick.

Reply:
ah yes that's right. Updating and merging.